server/src/main/java/org/elasticsearch/search/retriever/rankdoc/RankDocsQuery.java

```diff
@@ -260,13 +265,15 @@ public RankDocsQuery(IndexReader reader, RankDoc[] rankDocs, Query[] sources, St
             this.tailQuery = null;
         }
         this.onlyRankDocs = onlyRankDocs;
+        this.minScore = minScore;
     }
 
     private RankDocsQuery(RankDoc[] docs, Query topQuery, Query tailQuery, boolean onlyRankDocs) {
         this.docs = docs;
         this.topQuery = topQuery;
         this.tailQuery = tailQuery;
         this.onlyRankDocs = onlyRankDocs;
+        this.minScore = Float.MIN_VALUE;
```

Comment:
This should use a default member that you create and reference as suggested earlier. Alternately consider a convenience constructor that sets the default.